epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModelWithHolder.java

```diff
@@ -22,7 +24,7 @@ public EpoxyModelWithHolder(long id) {
   }
 
   /** This should return a new instance of your {@link com.airbnb.epoxy.EpoxyHolder} class. */
-  protected abstract T createNewHolder();
+  protected abstract T createNewHolder(@NonNull ViewParent parent);
```

Comment:
I suppose this would be a breaking change for a few people and they would have to update. I'm not opposed to that though

Reply:
Yes, you're right, I am not opposed too.